src/app/(dashboard)/review/[district]/[month]/page.tsx

```diff
@@ -0,0 +1,19 @@
+import { ReviewAssignmentPageClient } from '@/components/review/review-assignment';
+
+interface ReviewAssignmentPageProps {
+  params: {
+    district: string;
+    month: string;
+  };
+}
+
+export default function ReviewAssignmentPage({
```

Comment:
Maybe rename this? It isnt technically an assignment. I would call it "ReviewDashboardPage